doc/developing/index.rst

```diff
@@ -13,4 +13,6 @@ integration and more customisation options.
 .. toctree::
     :maxdepth: 1
 
+    environment
     api_references/modules
```

Comment:
@henriknorin-ftrack api_reference would be better suited on a higher level , so the develpment docs are separated from the api themselves (similar to our API)